yb-voyager/cmd/cutoverToTargetCmd.go

```diff
@@ -52,13 +52,14 @@ var cutoverToTargetCmd = &cobra.Command{
 			if msr.FallForwardEnabled {
 				utils.ErrExit("cannot prepare for fall-back. Fall-forward workflow already enabled.")
 			}
-			updateFallBackEnabledInMetaDB()
 		}
 		err = InitiateCutover("target")
 		if err != nil {
 			utils.ErrExit("failed to initiate cutover: %v", err)
 		}
-
+		if prepareForFallBack {
```

Comment:
I think the updation of record.CutoverToTargetRequested and record.FallBackEnabled should happen atomically. 

Theoretically, there is a possibility that marking record.CutoverToTargetRequested causes the exporter to shut down before record.FallBackEnabled is even set, since they are happening one after the other. (I think that's probably the reason I decided to mark the fallBackEnabled before). 
Practically, it's very unlikely, but I think it's better to still group these two operations together atomically.

Reply:
Yeah, makes sense. Made the change. thanks